no_drama/build.py

```diff
@@ -50,6 +50,18 @@ def stage_bundle(cli_args):
         project_destination = os.path.join(build_dir, project_slug)
         shutil.copytree(cli_args.project_path, project_destination)
 
+        # create a 'manage.py' script that runs in the virtualenv
+
+        manage_py_script = '''#!/bin/sh
+"exec" "`dirname $0`/venv/bin/python" "$0" "$@"
+
+import runpy
+runpy.run_path('%s/manage.py', run_name='__main__')
```

Comment:
In theory there's no guarantee that a manage.py script exists in the root of the Django project being built, right? In practice there is when run against cfgov-refresh, I suppose, but it's at least possible that this would be a mapping to a manage.py that doesn't exist.